index.html

```diff
@@ -0,0 +1,126 @@
+        <li class="contact-info">240-3169812</li>
+      </ul>
+    </header>
+
+    <main>
+      <section class="experience" id="experience">
+        <h2>Experience</h2>
+
+        <h3>Amazon - Warehouse Associate</h3>
+        <div>
+          <p>June 2019 - Present | Baltimore, MD.</p>
+          <p>
+            Description of the work
+          </p>
+        </div>
+
+        <h3>Fenton's Uniforms - Shipping Associate</h3>
+        <div>
+          <p>May 2018 - August 2019 | Silver Spring, MD.</p>
+          <p>
+            Description of the work
+          </p>
+        </div>
+
+        <h3>Cafritz Art Store - Key Holder</h3>
+        <div>
+          <p>October 2016 - February 2018 | Takoma Park, MD.</p>
+          <p>
+            Description of the work
+          </p>
+        </div>
+
+        <h3>Excalibur Student Newspaper - Layout Designer</h3>
+        <div>
+          <p>May 2016 - May 2018. | Silver Spring, MD.</p>
+          <p>
+            Description of the work
+          </p>
+        </div>
+
+        <h3>Excalibur Student Newspaper - Chief Editor</h3>
+        <div>
+          <p>May 2017 - May 2018. | Silver Spring, MD.</p>
+          <p>
+            Description of the work
+          </p>
+        </div>
+      </section>
+
+      <section id="education">
+        <h2>Education</h2>
+
+        <h3>Maryland Institute of College of Art</h3>
+        <div>
+          
+        <p>Baltimore, MD. 2019-Present. BFA in Graphic Design.</p></div>
+
+        <h3>Montgomery College</h3>
+       <div> <p>
+          Silver Spring, MD. 2016 - 2019. AFA in Graphic Design. Student
+          Leadership Recognition, 2017, 2018. Dean’s List 2016 - 2017.
+         </p></div>
+
+        <h3>Central University of Venezuela.</h3>
+       <div> <p>Caracas, Venezuela. 2012-2016. Incomplete. Architecture.</p>
+        </div> </section>
+
+      <--! cant make this sections one next to the other... cant make the h4 next to the description... ugh--> 
+      
+      <section class="detailed-info" id="exhibitions">
+        <h3>Exhibitions</h3>
+
+        <h4 >Liber Transformed</h4>
+        <div> <p>Baltimore MD, 2020.</p> </div>
+
+        <h4>Claiming Design</h4>
+        <div><p>Baltimore MD, 2020.</p></div>
+
+        <h4>Juried Group Exhibition</h4>
+      <div>
+        
+        <p>King Street Gallery, Takoma Park, MD, 2018.</p></div>
+      </section>
+
+      <section class="detailed-info">
+        <h3>Tools</h3>
+        <ul>
+          <li>Photoshop</li>
+          <li>Illustrator</li>
+          <li>InDesign</li>
+          <li>After Effects</li>
+          <li>Premiere</li>
+        </ul>
+      </section >
+    </main>
+
+    <footer>
+      <a class="footer" href="#aboutme">About Me</a>
+      <a class="footer" href="#experience">Experience</a>
+      <a class="footer" href="#education">Education</a>
+      <a class="footer" href="#exhibitions">Exhibitions </a>
```

Comment:
I think having an HTML element and a class with a shared name can lead to confusion. Maybe consider changing this class name to `footer-link` instead?